src/main/java/graphql/parser/GraphqlAntlrToLanguage.java

```diff
@@ -636,17 +631,45 @@ public Void visitInputObjectTypeDefinition(GraphqlParser.InputObjectTypeDefiniti
         return null;
     }
 
-    @Override
-    public Void visitTypeExtensionDefinition(GraphqlParser.TypeExtensionDefinitionContext ctx) {
-        TypeExtensionDefinition def = new TypeExtensionDefinition();
-        newNode(def, ctx);
+    private Void extensionTypeImpl(ParserRuleContext ctx, Definition def, ContextProperty ctxProperty) {
+        newNode((AbstractNode) def, ctx);
         result.getDefinitions().add(def);
-        addContextProperty(ContextProperty.TypeExtensionDefinition, def);
+        addContextProperty(ctxProperty, def);
         super.visitChildren(ctx);
         popContext();
         return null;
     }
 
+    @Override
```

Comment:
New naming in the .g4 generates new entry points.  But I think the name change is a little better

Reply:
👍